qt-models/CMakeLists.txt

```diff
@@ -1,31 +1,47 @@
 # the data models that will interface
 # with the views.
-set(SUBSURFACE_MODELS_LIB_SRCS
-	cleanertablemodel.cpp
-	cylindermodel.cpp
-	diveplannermodel.cpp
-	models.cpp
-	filtermodels.cpp
-	tankinfomodel.cpp
-	weigthsysteminfomodel.cpp
-	weightmodel.cpp
-	divecomputermodel.cpp
+
+# models used both mobile and desktop builds
+set(SUBSURFACE_GENERIC_MODELS_LIB_SRCS
+        cleanertablemodel.cpp
+        cylindermodel.cpp
+        models.cpp
+        tankinfomodel.cpp
+        divepicturemodel.cpp
+        diveplannermodel.cpp
 	treemodel.cpp
-	yearlystatisticsmodel.cpp
-	divetripmodel.cpp
-	divecomputerextradatamodel.cpp
-	completionmodels.cpp
-	divepicturemodel.cpp
-	diveplotdatamodel.cpp
-	divelocationmodel.cpp
-	ssrfsortfilterproxymodel.cpp
-	divelistmodel.cpp
-	gpslistmodel.cpp
-	diveimportedmodel.cpp
-	messagehandlermodel.cpp
-	maplocationmodel.cpp
+        diveplotdatamodel.cpp
+        diveimportedmodel.cpp
+)
+
+# models exclusively used in desktop builds
+set(SUBSURFACE_DESKTOP_MODELS_LIB_SRCS
+        maplocationmodel.cpp
+        yearlystatisticsmodel.cpp
+        weigthsysteminfomodel.cpp
+        weightmodel.cpp
+        filtermodels.cpp
+        divecomputermodel.cpp
+        divetripmodel.cpp
+        divecomputerextradatamodel.cpp
+        completionmodels.cpp
+        divelocationmodel.cpp
+        ssrfsortfilterproxymodel.cpp
+)
+
+# models exclusively used in mobile builds
+set(SUBSURFACE_MOBILE_MODELS_LIB_SRCS
+        divelistmodel.cpp
+        messagehandlermodel.cpp
+        gpslistmodel.cpp
 )
 
-source_group("Subsurface Models" FILES ${SUBSURFACE_MODELS})
```

Comment:
I don't recall if Qt Creator (or Xcode) respect "source_group" definitions, but we seem to have lost this

Reply:
I searched for SUBSURFACE_MODELS to see how it would expand ... but nothing found in cmake txt files. So just tested the delete on Travis, and no (obvious) differences.